support/deployNextFromTravis.sh

```diff
@@ -24,19 +24,22 @@ function mostRecentTag {
   echo $( git describe --tags --abbrev=0 $1)
 }
 
-echo "Determining build deployability 🔍"
+echo "Determining @next deployability 🔍"
 
 if [ ! $( deployable $( mostRecentTag HEAD ) ) ]
 then
   echo "No changes since the previous release, skipping ⛔"
 else
+  echo "Building deployable candidate ⚙️"
+  npm run build >/dev/null
```

Comment:
ok this is the magic place I was looking for, thank you @jcfranco 🙏🏻

Reply:
![giphy-2](https://user-images.githubusercontent.com/197440/115796957-9d4e5100-a387-11eb-809f-8575561dc861.gif)